miniapp/miniapp_triangular_multiplication.cpp

```diff
@@ -221,69 +236,3 @@ int main(int argc, char** argv) {
   p.rp_callback = dlaf::initResourcePartitionerHandler;
   return pika::init(pika_main, argc, argv, p);
 }
-
-namespace {
-/// Returns a tuple of element generators of three matrices A(m x m), B (m x n), X (m x n), for which it
-/// holds op(A) X = alpha B (alpha can be any value).
-///
-/// The elements of op(A) (@p el_op_a) are chosen such that:
-///   op(A)_ik = (i+1) / (k+.5) * exp(I*(2*i-k)) for the referenced elements
-///   op(A)_ik = -9.9 otherwise,
-/// where I = 0 for real types or I is the complex unit for complex types.
-///
-/// The elements of X (@p el_x) are computed as
-///   X_kj = (k+.5) / (j+2) * exp(I*(k+j)).
-/// These data are typically used to check whether the result of the equation
-/// performed with any algorithm is consistent with the computed values.
-///
-/// Finally, the elements of B (@p el_b) should be:
-/// B_ij = (Sum_k op(A)_ik * X_kj) / alpha
-///      = (op(A)_ii * X_ij + (kk-1) * gamma) / alpha,
-/// where gamma = (i+1) / (j+2) * exp(I*(2*i+j)),
-///       kk = i+1 if op(a) is an lower triangular matrix, or
-///       kk = m-i if op(a) is an lower triangular matrix.
-/// Therefore
-/// B_ij = (X_ij + (kk-1) * gamma) / alpha, if diag == Unit
-/// B_ij = kk * gamma / alpha, otherwise.
-template <typename T>
-linear_system_t<T> sampleLeftTr(blas::Uplo uplo, blas::Op op, blas::Diag diag, T alpha, SizeType m) {
```

Comment:
If this function is no longer needed, can we remove the forward declaration further up the file as well (including whatever typedefs etc. the declaration requires)?